components/content/index.css

```
@@ -1,21 +1,42 @@
 .content {
     background-color: var(--blueviolet);
+    border-radius: 12px;
 }
 
 .content-holder {
     display: flex;
     height: 100%;
-    background-image: linear-gradient(var(--red), var(--green));
+    border-radius: 12px;
 }
 
 /* Content aside */
 
 .content-aside {
-    width: 20%;
+    width: 40%;
     min-width: 100px;
-    background-image: url(../../assets/img/aside-background.jpg);
+    background-color: var( --light-gray-tint-1 );
+    display: flex;
+    flex-direction: column;
+}
```

```suggestion
    padding: 64px;
}
```

Add paddings for the entire aside element